src/index.js

```diff
@@ -1,124 +1,112 @@
-    } else {
-        console.warn('setDefaultNavigationButtonsVisibility is not available for iOS');
-    }
-}
-
-/**
- * This method will load the feedback with no screenshot attachement
- * @param {String} formId : feedback form Id
- */
-function loadFeedbackForm(formId) {
-    UsabillaBridge.loadFeedbackForm(formId)
-}
-
-/**
- * This method will load the feedback form and attach 
- * the current view screenshot to it.
- * @param {String} formId : feedback form Id
- */
-function loadFeedbackFormWithCurrentViewScreenshot(formId) {
-    UsabillaBridge.loadFeedbackFormWithCurrentViewScreenshot(formId)
-}
-
-function sendEvent(event) {
-    UsabillaBridge.sendEvent(event)
-}
-
-function removeCachedForms() {
-    UsabillaBridge.removeCachedForms()
-}
-
-function resetCampaignData(callback) {
-    if (Platform.OS == 'android') {
-        UsabillaBridge.resetCampaignData()
-    } else {
-        if (callback) {
-            UsabillaBridge.resetCampaignData(callback)
-            return
+import { NativeEventEmitter, NativeModules, Platform, DeviceEventEmitter } from 'react-native';
+const UsabillaBridge = NativeModules.UsabillaInterface;
+const usabillaEventEmitter = (Platform.OS == 'android') ? DeviceEventEmitter : new NativeEventEmitter(UsabillaBridge);
+
+export default {
+    
+    initialize(appId) {
+        if (Platform.OS == 'android') {
+            usabillaEventEmitter.addListener(
+                'UBFormNotFoundFragmentActivity',
+                () => console.log("The Activity does not extend FragmentActivity and cannot call getSupportFragmentManager()")
+            )
         }
-        UsabillaBridge.resetCampaignData(()=> {
-            console.log("Campaign data is successfully reset!")
-        })
-    }
-}
-
-function showLoadedForm() {
-    UsabillaBridge.showLoadedFrom()
-}
-
-function setCustomVariables(customVariables) {
-    UsabillaBridge.setCustomVariables(customVariables)
-}
-
-function setFormDidLoadSuccessfully(callback) {
-    usabillaEventEmitter.addListener('UBFormLoadingSucceeded', callback)
-}
-
-function setFormDidFailLoading(callback) {
-    usabillaEventEmitter.addListener('UBFormLoadingFailed', callback)
-}
-
-function setFormDidClose(callback) {
-    if (Platform.OS == 'ios') {
-        usabillaEventEmitter.addListener('UBFormDidClose', callback)
+        return UsabillaBridge.initialize(appId)
+    },
+    
+    areNavigationButtonsVisible() {
+        if (Platform.OS == 'android') {
+            return UsabillaBridge.areNavigationButtonsVisible()
+        } else {
+            console.warn('areNavigationButtonsVisible is not available for iOS');
+            return true
+        }
+    },
+    
+    setDefaultNavigationButtonsVisibility(visible) {
+        if (Platform.OS == 'android') {
+            return UsabillaBridge.setDefaultNavigationButtonsVisibility(visible)
+        } else {
+            console.warn('setDefaultNavigationButtonsVisibility is not available for iOS');
```

Comment:
Nothing to do with your PR:  Fascinating - how many differences are there like this between the platforms?